docs/source/platform/schema-registry.md

```diff
@@ -1,102 +1,102 @@
-title: Registering your schema
-description: How to publish your schema to the Apollo registry
+title: Track schema change history
+description: Connect to the Apollo schema registry and track changes in your schema over time
 ---
 
-A schema is the center point of all GraphQL applications. It powers incredible development tools, is an always up to date set of documentation, and creates an enforcable contract between clients and servers through validation. Since this is such a center point of how teams work on GraphQL, the Apollo GraphQL Platform provides a free schema registry for teams of all sizes to use. In storing a schema in the registry, teams can share an always up to date picture of their data model into every facet of their workflow.
+The [GraphQL schema](https://www.apollographql.com/docs/tutorial/schema.html) is the center point of all GraphQL applications. It creates an enforceable contract between clients and servers, it is its own self-updating API documentation, and it provides powerful development workflows thanks to its type safety. As GraphQL scales in an organization and as GraphQL schemas grow, it becomes really important to have tools to help your team safely evolve your schema over time. At scale, you might have hundreds or thousands of clients in the world making queries that rely on fields, types, and arguments in your schema that are impossible to track without tooling.
 
-<h2 id="benefits">Benefits of publishing a schema</h2>
+At Apollo, we're driven to help teams be successful with GraphQL both at the initial adoption phase and as they scale GraphQL across their organization. As such, the [Apollo Platform](/docs/intro/platform.html) provides a free schema registration service for teams of all sizes to use. Much like version control for your code, the Schema Registry allows you to evolve your API over time while keeping a record of when each change was made and helping you catch breaking changes before they're every deployed.
 
-The Apollo schema registry provides a number of benefits to help teams collaborate and ship high quality software faster. A few of these benefits are:
+<h4 id="benefits">Benefits of registering your schema</h4>
 
-* Powering editor tools like the [Apollo VS Code extension](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo)
-* Empowering better code reviews and safer changes with [schema validation](./schema-validation.html)
-* Having a single point of knowledge for all teams to view through the [Apollo schema explorer](https://engine.apollographql.com)
-* Sharing upcoming changes with [schema tags](#schema-tags)
-* Protecting the server along with the [Apollo operation registry](./operation-registry.html)
-* Having a historical view of how a schema changes with the [Apollo schema history tab in Engine](#history)
+By keeping your schema up-to-date in Apollo's schema registry, you can get a number of workflow benefits through integrations with Apollo's other tools:
```

Comment:
an example of without -ing: 

An up-to-date schema in Apollo's schema registry enables a number of workflow benefits through integrations with other tools. A registered schema combined with: 

- The [Apollo VS Code extension](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo) provides built-in linting on queries by validating against the schema in your registry, and annotates fields on your queries with performance indicators collected in Apollo's trace warehouse.
- [Schema validation](./schema-validation.html) creates a diff between your local schema and the last schema uploaded to the registry, and validates this diff against live traffic seen on your endpoint to make sure you never accidentally deploy a breaking schema change to production.
- The [Schema History](#history) keeps track of all the changes made to your schema over time.